assets/dev/js/frontend/frontend.js

```diff
@@ -313,6 +323,8 @@ export default class Frontend extends elementorModules.ViewModule {
 			this.muteMigrationTraces();
 		}
 
+		this.populateActiveBreakpointsConfig();
```

Comment:
It has to be done as early as possible

Reply:
Should I put it in the `Frontend` class constructor?